core/src/test/java/com/yahoo/oak/ValueUtilsSimpleTest.java

```diff
@@ -24,7 +24,7 @@ public void init() {
             new SyncRecycleMemoryManager(new NativeMemoryAllocator(128));
         s = syncRecycleMemoryManager.getEmptySlice();
         syncRecycleMemoryManager.allocate(s, 16, false);
-        s.getByteBuffer().putInt(s.getOffset(), 1);
+        UnsafeUtils.unsafe.putInt(s.getAddress(), 1);
```

Comment:
in this test slice is used for other stuff as well, 
Moreover if i want to change it i need it to be writebuffer, which has a private constructor 
all in all it will be messier

Reply:
OK